### ISSUE TITLE
Make instant or almost instant "fly" to the selected bookmark

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#8170]
A user complained that it's too slow. Considering that he is in Australia, there's little benefit for him to see the screen moving over an ocean.

**Comment by patepelo** [comment]
IMO it's ok now

**Comment by biodranik** [comment]
I think it depends and can be slow on some devices. Need to investigate first.

**Comment by Misalf-git** [comment]
What about changing the duration of the "flight" transition to not depend on its own movement speed but simply one second or some fraction of the distance?
**Additionally** just skip a certain distance (kilometers/miles) in the mid part of the "flight"?

It's good to have some movement in order to get a sense of direction but it's irritating to watch the map stuttering over the screen.

**Comment by muralito** [comment]
I checked with version 2024.03.31-8-FDroid, and for 10K Km (9942 to be precise), the movement first zooms out, then slide and then zoom in.

Maybe the slide could start simultaneously with the zoom out and end with the zoom in. Begin with constant acceleration, then in the middle a period of no acceleration, then constant deceleration until arrival, like if it was a flight.

**Comment by patepelo** [comment]
> Maybe the slide could start simultaneously with the zoom out and end with the zoom in. Begin with constant acceleration, then in the middle a period of no acceleration, then constant deceleration until arrival, like if it was a flight.

Yeah, maybe zoom out can be fast because it wouldn't matter to know origin of view port but zoom in should tell the user where it landed.
